modules/apps/object/object-test/src/testFunctional/tests/Picklist.testcase

```diff
@@ -73,6 +73,74 @@ definition {
 		Picklist.assertPicklistNotPresent(picklistName = "Update Picklist Test");
 	}
 
+	@description = "LPS-193364 - Verify the user can create a custom picklist and an object entry with an item from the picklist"
+	@priority = 4
+	test CanCreateObjectEntryWithPicklist {
+		task ("Given a custom picklist with several items") {
+			Picklist.addPicklistViaAPI(picklistName = "Picklist 193364");
+
+			for (var itemName : list "1,2,3,4,5") {
+				Picklist.addPicklistItemViaAPI(
+					itemKey = "item${itemName}",
+					itemName = "Item ${itemName}",
+					picklistName = "Picklist 193364");
+			}
+		}
+
+		task ("and Given the picklist has the custom label") {
+			Picklist.gotoPicklists();
+
+			AssertTextEquals(
+				key_label = "Custom",
+				locator1 = "ObjectAdmin#OBJECT_STATUS",
+				value1 = "Custom");
+		}
+
+		task ("and Given a custom object definition with a picklist field") {
+			ObjectAdmin.addObjectViaAPI(
+				labelName = "Custom Object 193364",
+				objectName = "CustomObject193364",
+				pluralLabelName = "Custom Objects 193364");
+
+			ObjectAdmin.openObjectAdmin();
+
+			ObjectPortlet.selectCustomObject(label = "Custom Object 193364");
+
+			ObjectAdmin.goToFieldsTab();
+
+			ObjectAdmin.addObjectFieldViaUI(
+				fieldLabel = "List Field",
+				fieldPicklist = "Picklist 193364",
+				fieldType = "Picklist",
+				itemName = "Item 3",
+				markAsState = "true");
+		}
+
+		task ("When the user creates an object entry") {
+			ObjectAdmin.publishObjectViaAPI(objectName = "CustomObject193364");
+
```

Comment:
The publish is part of the process to create the object definition, it should be in the previous task.